src/applications/personalization/profile-2/components/personal-information/PersonalInformation.jsx

```diff
@@ -1,26 +1,53 @@
-import React, { useEffect } from 'react';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Prompt } from 'react-router-dom';
+import { connect } from 'react-redux';
 
 import { focusElement } from 'platform/utilities/ui';
 
 import PersonalInformationContent from './PersonalInformationContent';
 
-const PersonalInformation = () => {
-  useEffect(() => {
-    focusElement('[data-focus-target]');
-  }, []);
-
-  return (
-    <>
-      <h2
-        tabIndex="-1"
-        className="vads-u-margin-y--2 medium-screen:vads-u-margin-bottom--4 medium-screen:vads-u-margin-top--3"
-        data-focus-target
-      >
-        Personal and contact information
-      </h2>
-      <PersonalInformationContent />
-    </>
-  );
+class PersonalInformation extends Component {
+  componentDidUpdate() {
```

Comment:
I bet we could maintain this as a functional component. But at the very least we need to maintain the `focusElement` call :)

Reply:
Encouraging me to learn about hooks, aren't you? 😄 I actually really appreciate that. I really gotta shake that familiarity bias! Hooks are pretty awesome after that initial learning curve - anyways, I believe I got it to work!